lib/puma/systemd.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'puma/events'
+
+module Puma
+  class Systemd
+    def initialize(events)
+      @events = events
+    end
+
+    def start_watchdog
+      usec = Integer(ENV["WATCHDOG_USEC"])
+      return log "systemd Watchdog too fast: " + usec if usec < 1_000_000
```

Comment:
I think we should warn rather than just log here, because the watchdog is effectively disabled by this method returning.